src/sass/_reviews.scss

```diff
@@ -0,0 +1,136 @@
+.reviews {
```

Comment:
Coś jest nie tak z wyświetlaniem przy 320px. W przeglądarce w inspektorze gdy ustawię stronę na 320px to pojawia mi się suwak poziomy i trzeba przesuwać żeby zobaczyć całą szerokość sekcji. Sprawdź czy jakiś element nie wyszedł ci poza container.

Reply:
Sprawdziłem właśnie, przy 320px nie mam paska poziomego, a każdy z osobna element ma max 320px. Co najlepsze te sekcje są tak napisane, że schodząc do 250px szerokości wszystko się ladnie zwęża bez paska.

Reply:
Poza container jest wstawiona tylko zmiana tła w sekcji gallery, aby już na przyszłość przy ekranie przekraczającym max-width strony, ładnie się tło rozciagało do granic ekranu.

Reply:
Jeszcze raz pobrałem twoją sekcję i już jest ok :) Dobra robota.

Reply:
Dzięki, jeśli byś mnie do czegoś innego potrzebował to mów, bo mam wolne do poniedzialku, a aktualnie biorę sie za JS.